Simplenote/src/main/java/com/automattic/simplenote/utils/BaseExt.kt

```diff
@@ -0,0 +1,21 @@
+package com.automattic.simplenote.utils
+
+import android.widget.Toast
+import androidx.annotation.ColorRes
+import androidx.annotation.StringRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+
+fun AppCompatActivity.toast(@StringRes resId: Int, length: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(this, getString(resId), length).show()
+}
+
+fun AppCompatActivity.getColorStr(@ColorRes color: Int): String {
+    val hintColor = ContextCompat.getColor(this, color)
+    val hintHexColor = hintColor.toHexString().replace("ff", "")
+    return "#$hintHexColor"
```

Comment:
Question (❓): Is this `#` character a new addition, it didn't exist before, right?

Reply:
Yes, I added it because if you get a color using `getString(...)`, it will include `#`, but since we are getting the color using `getColor(...)`, it does not include it.

Reply:
Ah, nice, thanks for the clarification Danilo! 🙏